sf-fx-runtime-java-logger/src/main/java/com/salesforce/functions/jvm/runtime/logger/DefaultLoggingFormatter.java

```diff
@@ -23,17 +25,32 @@ public DefaultLoggingFormatter(Clock clock) {
   public String format(String loggerName, Level level, String message) {
     String invocationId = MDC.get("function-invocation-id");
     String localDateTimeString = LocalDateTime.now(clock).format(DATE_TIME_FORMATTER);
+    String zoneId = clock.getZone().toString();
+    Map<String, String> stringFields =
+        new HashMap<String, String>() {
+          {
+            put(zoneId, localDateTimeString);
+            put("level", level.toString());
+            put("invocationId", invocationId);
+            put("loggerName", Utils.shortenLoggerName(loggerName, 36));
+            put("message", message);
+          }
+        };
 
-    return String.format(
-        FORMAT_STRING,
-        localDateTimeString,
-        level.toString(),
-        invocationId,
-        Utils.shortenLoggerName(loggerName, 36),
-        message);
+    String formattedString = formatString(stringFields);
+
+    return formattedString;
   }
 
-  private static final String FORMAT_STRING = "%s %-5s [INVOCATION %s] %s - %s\n";
+  private String formatString(Map<String, String> stringFields) {
+    StringBuilder formattedString = new StringBuilder();
+    for (String field : stringFields.keySet()) {
+      String currentFieldValue = String.format("\"%s\"=\"%s\" ", field, stringFields.get(field));
```

Comment:
This does not do any escaping. For details, see my comment below for the test that is supposed to test this. Please note that we also need to escape the escape character `\`.